portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+
+
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
```

Comment:
nit: add a javadoc for this servlet. See https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required: "At the minimum, Javadoc is present for every public class, and every public or protected member of such a class, with a few exceptions noted below"

Reply:
Any updates on this?

Reply:
Added